examples/plot_glm_retinal_ganglia.py

```diff
@@ -0,0 +1,421 @@
+# -*- coding: utf-8 -*-
+"""
+=================================================================
+GLM for Spike Trains Prediction in Primate Retinal Ganglion Cells
+=================================================================
+
+* Original tutorial adapted from Johnathan Pillow, Princeton University
+* Dataset provided by E.J. Chichilnisky, Stanford University
+* The dataset is granted by the original authors for the educational use in this tutorial
+* If you want to use it beyond educational purposes, please contact ``pillow@princeton.edu``
+
+First of all, we would like to thank Professor Johnathan Pillow and Professor E.J. Chichilnisky
+for sharing their tutorial dataset. The original MATLAB and Python tutorial can be found from
+https://github.com/pillowlab/GLMspiketraintutorial.
+
+These data were collected by Valerie Uzzell in the lab of
+E.J. Chichilnisky at the Salk Institute.  For full information see
+Uzzell et al. (J Neurophys 04), or Pillow et al. (J Neurosci 2005).
+**Again, the dataset in this tutorial is granted by the original authors
+for educational use only not for publication**. If you want to use it
+beyond the educational purposes,
+please contact ``pillow@princeton.edu``
```

Comment:
make this a prompt as I commented in the other PR :)

Reply:
Thanks! Can you put a reference to this from MNE @jasmainak?

Reply:
here you go: https://github.com/mne-tools/mne-python/blob/master/mne/datasets/utils.py#L379-L386

Reply:
Done! Let me know if it looks good to you!